doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_cpu.rst

```diff
@@ -168,6 +172,10 @@ System Controller
 *****************
 
 System Controller is a VPR that implements system startup and power management functionalities that in the past would have been implemented in hardware.
+The System Controller Firmware (SCFW) is part of the nRF54H20 SoC binaries.
```

Comment:
Here too I think we should relate this to IronSide SE if we are talking about the SoC binaries.